packages/unlock-js/examples/purchase-key.js

```diff
@@ -3,7 +3,7 @@ const ethers = require('ethers')
 const { WalletService } = require('../lib/index')
 const networks = require('./networks')
 
-const provider = new ethers.providers.JsonRpcBatchProvider(networks[4].provider)
```

Comment:
The batch provider was pretty cool as it made the app a bit faster by "bundling" multiple RPC calls into a single one. Is that not a thing anymore?

Reply:
the batch mode is actually enabled by default in ethers v6 https://docs.ethers.org/v6/api/providers/jsonrpc/#JsonRpcApiProviderOptions

default options are 

https://github.com/ethers-io/ethers.js/blob/92761872198cf6c9334570da3d110bca2bafa641/src.ts/providers/provider-jsonrpc.ts#L205-L207

Reply:
Very cool!